v2/jwt/jwt.go

```diff
@@ -89,6 +89,13 @@ func Parse(jwtToken string) (_ Token, err error) {
 		},
 	)
 	if err != nil {
+		if ve, ok := err.(*jwt.ValidationError); ok {
```

Comment:
Use `errors.As` instead?

Reply:
This was copy paste from the example in "github.com/golang-jwt/jwt/v4"